### PR TITLE
Update creating and updating connection instance cmdlets by adding one parameter

### PR DESCRIPTION
**Pull request by gucsun**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

**Review by get-itips — APPROVED** [review]

File: teams/teams-ps/teams/New-CsTeamsShiftsConnectionInstance.md [hunk 1]
```diff
@@ -299,6 +303,22 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -ConnectorAdminsEmail
+
+The list of connector admin email addresses that receive result notification
```
Comment (dariomws) [inline teams/teams-ps/teams/New-CsTeamsShiftsConnectionInstance.md:308]:
```suggestion
The list of connector admin email addresses that receive result notifications.
```

File: teams/teams-ps/teams/Set-CsTeamsShiftsConnectionInstance.md [hunk 2]
```diff
@@ -331,6 +336,22 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -ConnectorAdminsEmail
+
+The list of connector admin email addresses that receive result notification
```
Comment (dariomws) [inline teams/teams-ps/teams/Set-CsTeamsShiftsConnectionInstance.md:341]:
```suggestion
The list of connector admin email addresses that receive result notifications.
```

**Review by dariomws — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**